python/ray/train/v2/tests/test_async_checkpointing_validation.py

```diff
@@ -365,6 +367,105 @@ def train_fn():
     }
 
 
+def test_get_all_reported_checkpoints_all_checkpoint_views():
+    @ray.remote
+    class StateActor:
```

Comment:
can we use some dummy marker files instead of this extra actor? for a more minimal test

Reply:
Leaving as is since they recommended signal actor pattern.